docs/input/docs/learn/how-it-works.md

```diff
@@ -19,51 +19,54 @@ GitVersion has three distinct steps for calculating versions in v3.
 
-[Edit Diagram](https://www.plantuml.com/plantuml/form?url=https://www.plantuml.com/plantuml/png/fLCxJyCm4DxzAsuib4P914i69De1CS38Vd6kYIN7ZcodK8aVp-KX6Y2fKCbY9NV-7lVb2WoOeoVOMRDNfH0lz1vUoNbbpGwrR3K6ws1p3rlk-bN8u972f2AC3GHEbLN8m1D1Jjg-mPuXAZvx9kL1ZW1KY5dOZczMI0Pf54VnHtf7jpaAWJg0sW-uXw4PK3Eb1sMaevfCW6i1\_0m6po1l7HfPJUxvu5XYUOHLWq5MLptCudmMK9--u5glJ0dIEaVo1Dw3JgVM6Km4cM9mzyrQXHuQHnj7chhl0JcnIrHjno1wiWtgfi8eWVK\_7OQAmBHrJWvORFVM2PmrE7AcWZGh-Lj0FvptVvLiUPnCdG_XhNhOov9wQ1fzv7nw5S5EwSvw6CDQNfnMwUAP0XQyQpj70nkx3Nn3p5NFY9IshbNWepKi8ublWFiSPkC0ee8El75Dv5aOxqZQBScbWpWn0Pe2wb6aM1p4Eea\_0G00)
+[Edit Diagram](https://www.plantuml.com/plantuml/uml/fLCxJyCm4DxzAsuib4P914i69De1CS38Vd6kYIN7ZcodK8aVp-KX6Y2fKCbY9NV-7lVb2WoOeoVOMRDNfH0lz1vUoNbbpGwrR3K6ws1p3rlk-bN8u972f2AC3GHEbLN8m1D1Jjg-mPuXAZvx9kL1ZW1KY5dOZczMI0Pf54VnHtf7jpaAWJg0sW-uXw4PK3Eb1sMaevfCW6i1_0m6po1l7HfPJUxvu5XYUOHLWq5MLptCudmMK9--u5glJ0dIEaVo1Dw3JgVM6Km4cM9mzyrQXHuQHnj7chhl0JcnIrHjno1wiWtgfi8eWVK_7OQAmBHrJWvORFVM2PmrE7AcWZGh-Lj0FvptVvLiUPnCdG_XhNhOov9wQ1fzv7nw5S5EwSvw6CDQNfnMwUAP0XQyQpj70nkx3Nn3p5NFY9IshbNWepKi8ublWFiSPkC0ee8El75Dv5aOxqZQBScbWpWn0Pe2wb6aM1p4Eea_0G00)
 
 **\*** Some strategies allow the version to be incremented, others don't. More
 info below.
 **+** This version is out of context with the rest of the example. It is here
 simply to show what happens if the check is true.
 
-### Base Version Strategies
+### Version Strategies
 
 Currently we have the following strategies:
 
-* `HighestTagBaseVersionStrategy` - Finds the highest reachable tag from the
-    current branch
-* `VersionInBranchBaseVersionStrategy` - Extracts version information from the
+* `TaggedCommit` - Extracts version information from all tags on the branch which are valid,
+    and not newer than the current commit.
+* `VersionInBranchName` - Extracts version information from the
     branch name (e.g., `release/3.0.0` will find `3.0.0`)
-* `ConfigNextVersionBaseVersionStrategy` - Returns the version from the
+* `ConfiguredNextVersion` - Returns the version from the
     GitVersion.yaml file
-* `MergeMessageBaseVersionStrategy` - Finds version numbers from merge messages
+* `MergeMessage` - Finds version numbers from merge messages
     (e.g., `Merge 'release/3.0.0' into 'main'` will return `3.0.0`)
-* `FallbackBaseVersionStrategy` - Always returns 0.0.0 and will be used for
+* `Fallback` - Always returns 0.0.0 and will be used for
     calculating the next version which is dependent on the increment strategy of
  the effected branch (e.g. on main the next version is 0.0.1 or on develop it is 0.1.0)
+* `TrackReleaseBranches` - Considers the base version extracted from release branches when
+calculating the next version for branches configured with `track-release-branches: true`
+(part of default configuration for `develop` branch in `GitFlow` workflow)
+* `Mainline` - Increments the version on every commit for branches configured with `mode: Mainline`
```

Comment:
mode: Mainline is not valid anymore. I think is-main-branch: true would be better

Reply:
Understood! 
And fixed!